scripts/f2grfn_standalone.py

```diff
@@ -45,7 +44,7 @@
         "--directory",
         nargs="*",
         help="A temporary directory for generated files to be stored.",
-        default=".",
+        default=["."],
```

Comment:
@hlim1 I see now why this change is necessary for the script to work - but since we are only going to specify one temporary directory, I suggest removing the 'nargs="*" line (line 45) and reverting back to `default="."`.

Reply:
@adarshp I've reflected your suggestion and removed `nargs="*"` and reverted back to `default="."` and `args.directory`.